socorro/external/postgresql/graphics_report.py

```diff
@@ -57,7 +57,7 @@
         %(prod_phrase)s %(ver_phrase)s
       order by 5 -- r.date_processed, munged
 
-"""
+"""  # noqa
```

Comment:
actually, never mind. This is [going away](https://github.com/mozilla-services/socorro/pull/3831/files) soon.